packages/core/src/__tests__/ceramic-api.test.ts

```diff
@@ -377,74 +377,6 @@ describe('Ceramic API', () => {
       await ceramic.close()
     })
 
-    it('update schema so existing doc no longer conforms', async () => {
-      ceramic = await createCeramic()
-
-      const controller = ceramic.context.did.id
-
-      // Create doc with content that has type 'string'.
-      const tileDocParams: TileParams = {
-        metadata: {
-          controllers: [controller]
-        },
-        content: { a: 'x' },
-      }
-      const doc = await ceramic.createDocument<TileDoctype>(DOCTYPE_TILE, tileDocParams)
-      await syncDoc(doc)
-
-      // Create schema that enforces that the content value is a string
-      const schemaDoc = await ceramic.createDocument<TileDoctype>(DOCTYPE_TILE, {
-        content: stringMapSchema,
-        metadata: { controllers: [controller] }
-      })
-      // wait for anchor
-      await syncDoc(schemaDoc)
-      expect(schemaDoc.state.anchorStatus).toEqual(AnchorStatus.ANCHORED)
-      const schemaV0Id = DocID.fromBytes(schemaDoc.id.bytes, schemaDoc.tip.toString())
-
-      // Assign the schema to the conforming document.
-      await doc.change({
-        metadata: {
-          controllers: [controller], schema: schemaDoc.id.toString()
-        }
-      })
-      await syncDoc(doc)
-      expect(doc.content).toEqual({ a: 'x' })
-
-      // Update schema so that existing doc no longer conforms
-      const updatedSchema = cloneDeep(stringMapSchema)
-      updatedSchema.additionalProperties.type = "number"
-      await schemaDoc.change({content: updatedSchema})
-      await syncDoc(schemaDoc)
-
-      // Test that we can load the existing document without issue
-      const doc2 = await ceramic.loadDocument(doc.id)
```

Comment:
The whole point of this test was to test the behavior when loading a document that doesn't conform to its schema, but this check here isn't actually telling us anything useful since it's not going through the "real" document loading process, but instead just fetching it out of the local cache [here](https://github.com/ceramicnetwork/js-ceramic/blob/28da783f6c6ca29dc925152ce4c3c5fc6e3c0bdd/packages/core/src/ceramic.ts#L368).

To do this test properly it would need to be re-written as an integration test with two ceramic instances, one that creates the non-conforming document and a second that loads it.  I'm happy to do that if you think it's worthwhile to add, but in the interest of time I decided to just focus on unit testing the core Document/applyRecord behavior and trust that it continues to work when being driven by the pubsub network

Reply:
feel free to add a test into `ceramic.test.ts` test suite which is already spinning a couple of instances (ipfs+ceramic). It currently serves as a **basic** integration test for testing propagation between multiple instances. On the other side, it's not crucial to add it as part of this PR, we can open another story for that 👍